package.json

```diff
@@ -3,6 +3,11 @@
   "version": "1.0.2",
```

Comment:
We would need to bump the version, or is @UlisesGascon who will do it?

Reply:
In this case.. I will suggest to be included in the PR @Betisman @gentooXativa